Cannot build abp template project for linux-x64

I try to build apb template project for Linux but failed. 
Steps:
* Use the latest abp CLI v2.4 to create a new project:
```
abp new AbpWeb -u angular --separate-identity-server
```
* Build the `HttpApi.Host` project by this command:
```
dotnet build -r linux-x64
```

The output is:
```
  Restore completed in 20.72 sec for /home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application.Contracts/AbpWeb.Application.Contracts.csproj.
  Restore completed in 20.72 sec for /home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain.Shared/AbpWeb.Domain.Shared.csproj.
  Restore completed in 20.77 sec for /home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi/AbpWeb.HttpApi.csproj.
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605: Detected package downgrade: System.IO.FileSystem.Primitives from 4.3.0 to 4.0.1. Reference the package directly from the project to select a different version.  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605:  AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> runtime.unix.System.IO.FileSystem 4.3.0 -> System.IO.FileSystem.Primitives (>= 4.3.0)  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605:  AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> System.IO.FileSystem.Primitives (>= 4.0.1) [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605: Detected package downgrade: System.Runtime.Handles from 4.3.0 to 4.0.1. Reference the package directly from the project to select a different version.  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605:  AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> runtime.unix.System.IO.FileSystem 4.3.0 -> System.Runtime.Handles (>= 4.3.0)  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605:  AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> System.Runtime.Handles (>= 4.0.1) [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605: Detected package downgrade: System.Runtime.InteropServices from 4.3.0 to 4.1.0. Reference the package directly from the project to select a different version.  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605:  AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> runtime.unix.System.IO.FileSystem 4.3.0 -> System.Runtime.InteropServices (>= 4.3.0)  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605:  AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.Runtime.InteropServices (>= 4.1.0) [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
  Restore failed in 22.54 sec for /home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj.
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605: Detected package downgrade: System.IO.FileSystem.Primitives from 4.3.0 to 4.0.1. Reference the package directly from the project to select a different version.  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605:  AbpWeb.Application -> AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> runtime.unix.System.IO.FileSystem 4.3.0 -> System.IO.FileSystem.Primitives (>= 4.3.0)  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605:  AbpWeb.Application -> AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> System.IO.FileSystem.Primitives (>= 4.0.1) [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605: Detected package downgrade: System.Runtime.Handles from 4.3.0 to 4.0.1. Reference the package directly from the project to select a different version.  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605:  AbpWeb.Application -> AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> runtime.unix.System.IO.FileSystem 4.3.0 -> System.Runtime.Handles (>= 4.3.0)  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605:  AbpWeb.Application -> AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> System.Runtime.Handles (>= 4.0.1) [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605: Detected package downgrade: System.Runtime.InteropServices from 4.3.0 to 4.1.0. Reference the package directly from the project to select a different version.  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605:  AbpWeb.Application -> AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> runtime.unix.System.IO.FileSystem 4.3.0 -> System.Runtime.InteropServices (>= 4.3.0)  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605:  AbpWeb.Application -> AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.Runtime.InteropServices (>= 4.1.0) [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
  Restore failed in 22.93 sec for /home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj.
  Restore completed in 30.3 sec for /home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.EntityFrameworkCore.DbMigrations/AbpWeb.EntityFrameworkCore.DbMigrations.csproj.
  Restore completed in 30.3 sec for /home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.EntityFrameworkCore/AbpWeb.EntityFrameworkCore.csproj.
  Restore completed in 45.88 sec for /home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj.

Build FAILED.

/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605: Detected package downgrade: System.IO.FileSystem.Primitives from 4.3.0 to 4.0.1. Reference the package directly from the project to select a different version.  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605:  AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> runtime.unix.System.IO.FileSystem 4.3.0 -> System.IO.FileSystem.Primitives (>= 4.3.0)  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605:  AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> System.IO.FileSystem.Primitives (>= 4.0.1) [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605: Detected package downgrade: System.Runtime.Handles from 4.3.0 to 4.0.1. Reference the package directly from the project to select a different version.  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605:  AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> runtime.unix.System.IO.FileSystem 4.3.0 -> System.Runtime.Handles (>= 4.3.0)  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605:  AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> System.Runtime.Handles (>= 4.0.1) [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605: Detected package downgrade: System.Runtime.InteropServices from 4.3.0 to 4.1.0. Reference the package directly from the project to select a different version.  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605:  AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> runtime.unix.System.IO.FileSystem 4.3.0 -> System.Runtime.InteropServices (>= 4.3.0)  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Domain/AbpWeb.Domain.csproj : error NU1605:  AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.Runtime.InteropServices (>= 4.1.0) [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605: Detected package downgrade: System.IO.FileSystem.Primitives from 4.3.0 to 4.0.1. Reference the package directly from the project to select a different version.  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605:  AbpWeb.Application -> AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> runtime.unix.System.IO.FileSystem 4.3.0 -> System.IO.FileSystem.Primitives (>= 4.3.0)  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605:  AbpWeb.Application -> AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> System.IO.FileSystem.Primitives (>= 4.0.1) [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605: Detected package downgrade: System.Runtime.Handles from 4.3.0 to 4.0.1. Reference the package directly from the project to select a different version.  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605:  AbpWeb.Application -> AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> runtime.unix.System.IO.FileSystem 4.3.0 -> System.Runtime.Handles (>= 4.3.0)  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605:  AbpWeb.Application -> AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> System.Runtime.Handles (>= 4.0.1) [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605: Detected package downgrade: System.Runtime.InteropServices from 4.3.0 to 4.1.0. Reference the package directly from the project to select a different version.  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605:  AbpWeb.Application -> AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.IO.FileSystem 4.0.1 -> runtime.unix.System.IO.FileSystem 4.3.0 -> System.Runtime.InteropServices (>= 4.3.0)  [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.Application/AbpWeb.Application.csproj : error NU1605:  AbpWeb.Application -> AbpWeb.Domain -> Microsoft.AspNetCore.Mvc.DataAnnotations 2.2.0 -> Microsoft.AspNetCore.Mvc.Core 2.2.0 -> Microsoft.Extensions.DependencyModel 2.1.0 -> Microsoft.DotNet.PlatformAbstractions 2.1.0 -> System.Runtime.InteropServices (>= 4.1.0) [/home/myname/abpweb/AbpWeb/aspnet-core/src/AbpWeb.HttpApi.Host/AbpWeb.HttpApi.Host.csproj]
    0 Warning(s)
    6 Error(s)

```

I have installed .NET Core 3.1 SDK. I also tried to create the new project in WSL2 but got the same error. Any ideas? Thanks.

--- a/templates/app/aspnet-core/src/MyCompanyName.MyProjectName.Domain/MyCompanyName.MyProjectName.Domain.csproj
+++ b/templates/app/aspnet-core/src/MyCompanyName.MyProjectName.Domain/MyCompanyName.MyProjectName.Domain.csproj
@@ -12,7 +12,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.DataAnnotations" Version="2.2.0" />
     <ProjectReference Include="..\..\..\..\..\modules\identity\src\Volo.Abp.Identity.Domain\Volo.Abp.Identity.Domain.csproj" />
     <ProjectReference Include="..\..\..\..\..\modules\identity\src\Volo.Abp.PermissionManagement.Domain.Identity\Volo.Abp.PermissionManagement.Domain.Identity.csproj" />
     <ProjectReference Include="..\..\..\..\..\modules\identityserver\src\Volo.Abp.IdentityServer.Domain\Volo.Abp.IdentityServer.Domain.csproj" />

